jme3-bullet/src/common/java/com/jme3/bullet/control/RigidBodyControl.java

```diff
@@ -266,16 +269,20 @@ public void update(float tpf) {
     public void render(RenderManager rm, ViewPort vp) {
     }
 
-    public void setPhysicsSpace(PhysicsSpace space) {
+    public void setPhysicsSpace(final PhysicsSpace space) {
+        final PhysicsSpace currentSpace = getPhysicsSpace();
```

Comment:
I don't mean to sound ungrateful.  I appreciate the main fix.  It just wastes so much time to include other random stuff in what would have been a one line change otherwise.